### PR TITLE
EWPP-1148: Adapt site header, page header and breacrumbs to ecl V3

### PR DESCRIPTION
**Pull request by imanoleguskiza**
### Description

Adapt site header, page header and breacrumbs to ecl V3
### Change log

- Added:
- Changed: Adapt site header, page header and breacrumbs to ecl V3
- Deprecated:
- Removed:
- Fixed:
- Security:

### Commands

```sh
[Insert commands here]

```

### CODE REVIEW

File: templates/patterns/page_header/pattern-page-header.html.twig [hunk 1]
```diff
@@ -52,12 +46,12 @@
 
 {% include '@ecl-twig/page-header-' ~ ecl_branding ~ '/page-header-' ~ ecl_branding with {
   'title': title,
+  'thumbnail': thumbnail,
   'description': introduction,
   'breadcrumb': _breadcrumb,
-  'meta': _metas,
-  'infos': _infos,
-  'background_image': ecl_branding == 'core' and background_image_url is not empty,
+  'meta': metas,
```
Comment (sergepavle) [inline templates/patterns/page_header/pattern-page-header.html.twig]: I think we have to keep using 'filter_empty' filter.
```suggestion
  'meta': metas|filter_empty,
```

File: templates/patterns/page_header/pattern-page-header.html.twig [hunk 2]
```diff
@@ -52,12 +46,12 @@
...
   'background_image_url': background_image_url,
   'composition': _language_switcher,
   'extra_classes': _extra_classes|join(' '),
+  'overlay': overlay,
```
Comment (sergepavle) [inline templates/patterns/page_header/pattern-page-header.html.twig]: We have to provide this parameter only for EC ecl branding.

File: templates/compositions/ec-component-language-list/language-list-selector.html.twig [hunk 3]
```diff
@@ -66,7 +66,7 @@
 {% endif %}
 
 <a
-  class="ecl-link ecl-link--standalone ecl-site-header-{{ _ecl_branding }}__language-selector"
+  class="ecl-button ecl-button--ghost  ecl-site-header-{{ _ecl_branding }}__language-selector"
```
Comment (sergepavle) [inline templates/compositions/ec-component-language-list/language-list-selector.html.twig:69]: in templates/blocks/block--oe-search.html.twig classes seems like also should be updated.

**Review by sergepavle — CHANGES_REQUESTED** [review]

File: templates/compositions/ec-component-language-list/language-list-selector.html.twig (hunk 3 above)
Comment (imanoleguskiza) [reply templates/compositions/ec-component-language-list/language-list-selector.html.twig:69]: I took a look and I didn't find anything to be changed. Could you be more precise?

**Review by sergepavle — DISMISSED** [review]

**Review by ademarco — CHANGES_REQUESTED** [review]
Page header on EU / Core looks like below: we need to make sure both text and icon are visible and that the dropdown works correctly:

image

On EC / Core we can have trouble seeing the text, if a picture is below it:

image

**Comment by imanoleguskiza** [comment]
I pushed a fix for the colors in ec/eu flavors and to make sure nothing is invisible, but I am not sure how you propose we fix the issue with some of the text not being visibile when there is an image in the background, I feel like this is an issue editors need to take into account when uploading images.

**Comment by ademarco** [comment]
I've just got new reqs that we will now have a dedicated composition for the content language switcher, so we can discard fixing it in this PR. I'll open a follow up for that only.

**Review by ademarco — APPROVED** [review]

**Review by 22Alexandra — APPROVED** [review]

**Pull request merged**